### PR TITLE
Azure - Node - Machine Learning modules

### PR DESCRIPTION
**Pull request by jawache**
Updated management code sample

#sign-off as per @TomArcher

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/machine-learning.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-4/docs-ref-services/machine-learning.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C12%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707121735041910-72%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/machine-learning.md [hunk 1]
```diff
@@ -18,16 +18,36 @@ ms.service: Machine Learning
 ## Overview
 Machine learning is a technique of data science that helps computers learn from existing data in order to forecast future behaviors, outcomes, and trends. These forecasts or predictions from machine learning can make apps and devices smarter. When you shop online, machine learning helps recommend other products you might like based on what you've purchased. When your credit card is swiped, machine learning compares the transaction to a database of transactions and helps detect fraud. When your robot vacuum cleaner vacuums a room, machine learning helps it decide whether the job is done.
 
-## Install the modules with npm
+## Management
```
Comment (johnpapa) [inline docs-ref-services/machine-learning.md]: I think we went with `Management Package`

File: docs-ref-services/machine-learning.md [hunk 2]
```diff
@@ -18,16 +18,36 @@ ms.service: Machine Learning
 ## Overview
 Machine learning is a technique of data science that helps computers learn from existing data in order to forecast future behaviors, outcomes, and trends. These forecasts or predictions from machine learning can make apps and devices smarter. When you shop online, machine learning helps recommend other products you might like based on what you've purchased. When your credit card is swiped, machine learning compares the transaction to a database of transactions and helps detect fraud. When your robot vacuum cleaner vacuums a room, machine learning helps it decide whether the job is done.
 
-## Install the modules with npm
+## Management
 
 Use npm to install the Azure Machine Learning modules for Node.js
 
-### Management
+### Install the modules with npm
```
Comment (johnpapa) [inline docs-ref-services/machine-learning.md]: `Install the npm modules`

File: docs-ref-services/machine-learning.md [hunk 3]
```diff
@@ -18,16 +18,36 @@ ms.service: Machine Learning
...
-##Samples
+msRestAzure
+  .interactiveLogin()
+  .then(credentials => {
+    const subscriptionId = 'your-subscription-id';
+    const client = new MachineLearningManagement.CommitmentPlansManagementClient(
+      credentials,
+      subscriptionId
+    );
+    return client.commitmentPlans.list();
```
Comment (johnpapa) [inline docs-ref-services/machine-learning.md]: is this a promise? if so, this is fine. if not, then i woudl combine this code with the following `then`

File: docs-ref-services/machine-learning.md [hunk 4]
```diff
@@ -18,16 +18,36 @@ ms.service: Machine Learning
...
+  })
+  .then(commitmentPlans => {
+    console.log('List of commitmentPlans:');
+    console.dir(commitmentPlans, { depth: null, colors: true });
+    return;
```
Comment (johnpapa) [inline docs-ref-services/machine-learning.md]: do we need the `return`?

File: docs-ref-services/machine-learning.md [hunk 5]
```diff
@@ -18,16 +18,36 @@ ms.service: Machine Learning
...
+  });
+```
+
+### Samples
```
Comment (johnpapa) [inline docs-ref-services/machine-learning.md]: just confirming ... there are no samples for this?

File: docs-ref-services/machine-learning.md (hunk 3 above)
Comment (jawache) [inline docs-ref-services/machine-learning.md]: Yup it's a promise.

File: docs-ref-services/machine-learning.md (hunk 4 above)
Comment (jawache) [inline docs-ref-services/machine-learning.md]: No.. no we do not!

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/machine-learning.md](https://github.com/jawache/azure-docs-sdk-node/blob/patch-4/docs-ref-services/machine-learning.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C12%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707121805410964-72%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by johnpapa** [comment]
LGTM

**Comment by TomArcher** [comment]
@jawache Merged

**Pull request merged**